docs/source/gallery/examples/plotting/parallel.py

```diff
@@ -8,6 +8,7 @@
 """
 import matplotlib.axes
 import matplotlib.pyplot as plt
+from mpl_toolkits.axes_grid1 import make_axes_locatable
```

Comment:
For this example, the main change that is required (other than the `color` argument to `parallel()`) is maybe to specify the axes to be consumed (this should at least avoid the deprecation warning), i.e.:
```python
plt.colorbar(sm, ax=ax)
```
As written it's more verbose but more customizable, happy to leave as-is, it's good to be able to point to different ways to achieve/customize some of these `matplotlib` elements.

Reply:
Thanks for the head-up on the alternative change! 
I think I marginally prefer the added flexibility, so I'll leave this one as-is for now, but I'm not against the alternative.